### PR TITLE
LW-7691 upgrade node and ogmios

### PR DESCRIPTION
**Pull request by mirceahasegan**
# Context

Upgrade to node 8.9.1 and ogmios 6.2.0

### CODE REVIEW

**Review by mkazlauskas — APPROVED** [review]
LGTM :rocket:

File: compose/common.yml [hunk 1]
```diff
@@ -162,7 +162,7 @@ services:
 
   ogmios:
     <<: *logging
-    image: cardanosolutions/ogmios:v${OGMIOS_VERSION:-6.1.0}
+    image: cardanosolutions/ogmios:v${OGMIOS_VERSION:-6.2.0}
```
Comment (michalrus) [inline compose/common.yml:165]: 📝 I'll need to build this one for ARM, too #1172. Node 8.9.1 is already there.

**Review by michalrus — APPROVED** [review]
LGTM, thank you!

**Review by iccicci — APPROVED** [review]
Very nice!

Now #1174 was merged, I suggest to rebase this on top of new `conway-era` branch.

File: compose/common.yml (hunk 1 above)
Comment (michalrus) [reply compose/common.yml:165]: Done:
* [cardano-node:8.9.1](https://github.com/input-output-hk/ogmios-tracker/pkgs/container/ogmios-tracker%2Fcardano-node/197101568?tag=8.9.1)
* [ogmios:v6.2.0](https://github.com/input-output-hk/ogmios-tracker/pkgs/container/ogmios-tracker%2Fogmios/197122358?tag=v6.2.0)

**Pull request merged**